src/screens/CurrencySelectModal.js

```diff
@@ -257,9 +261,39 @@ export default function CurrencySelectModal() {
       type,
     ]
   );
+  const checkForRequiredAssets = useCallback(
+    item => {
+      if (
+        type === CurrencySelectionTypes.output &&
+        currentChainId &&
+        currentChainId !== ChainId.mainnet
+      ) {
+        const currentL2Name = ethereumUtils.getNetworkNameFromChainId(
+          currentChainId
+        );
+        const currentL2WalletAssets = assetsInWallet.filter(
+          ({ network }) =>
+            network && network?.toLowerCase() === currentL2Name?.toLowerCase()
+        );
+        if (currentL2WalletAssets?.length < 1) {
+          navigate(Routes.EXPLAIN_SHEET, {
```

Comment:
can we add 

```
    android && Keyboard.dismiss();
```

in android the sheet is being opened but the keyboard stays on top